connection/kafka_http_sideline_conn.go

```diff
@@ -0,0 +1,57 @@
+package connection
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/Shopify/sarama"
+	"github.com/flipkart-incubator/go-dmux/core"
+	sink "github.com/flipkart-incubator/go-dmux/http"
+	source "github.com/flipkart-incubator/go-dmux/kafka"
+	"github.com/flipkart-incubator/go-dmux/offset_monitor"
+	"log"
+	"os"
+)
+
+// **************** CONFIG ***********
+
+//KafkaHTTPWithSidelineConn struct to abstract this connections Run
+type KafkaHTTPWithSidelineConn struct {
+	EnableDebugLog bool
+	Conf           interface{}
+	SidelineImpl   interface{}
```

Comment:
Use proper types instead of open `interface{}`

Reply:
I think we don't need explicit connection for sideline. it can simply be a capability can be enabled/disabled by either http/foxtrot clients. we should ideally remove sidleline connection implementation.